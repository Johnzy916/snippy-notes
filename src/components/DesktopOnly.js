import React from 'react';

const DesktopOnly = () => {
    // overlay for when the screen is less than 550px
    // used for MVP as app is intended for desktop
    // would need to cover tracking keypress for Android and others
    // and tighten up media query design for mobile
    // -- can be found in PrivateRoute and AdminRoute
    return (
        <div className="mobile-not-available">
            This app was built for use on a desktop!
            <div className="desktop-overlay-img"></div>
            If you're on a desktop, please widen the screen.
        </div>
    );
}

export default DesktopOnly;