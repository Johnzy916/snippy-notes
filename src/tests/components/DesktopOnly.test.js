import React from 'react'
import { shallow } from 'enzyme'
import DesktopOnly from '../../components/DesktopOnly'

it('should render DesktopOnly correctly', () => {
    const wrapper = shallow(<DesktopOnly />)
    expect(wrapper).toMatchSnapshot()
});