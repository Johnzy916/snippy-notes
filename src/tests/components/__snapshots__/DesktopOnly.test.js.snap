// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DesktopOnly correctly 1`] = `
<div
  className="mobile-not-available"
>
  This app was built for use on a desktop!
  <div
    className="desktop-overlay-img"
  />
  If you're on a desktop, please widen the screen.
</div>
`;
